lib/vmdb/loggers/audit_logger.rb

```diff
@@ -1,13 +1,15 @@
-module Vmdb::Loggers
-  class AuditLogger < VMDBLogger
-    def success(msg)
-      info("Success") { msg }
-      $log.info("<AuditSuccess> #{msg}") if $log
-    end
+module Vmdb
+  module Loggers
+    class AuditLogger < VMDBLogger
+      def success(msg)
+        info("Success") { msg }
+        $log.info("<AuditSuccess> #{msg}") if $log
+      end
 
-    def failure(msg)
-      warn("Failure") { msg }
-      $log.warn("<AuditFailure> #{msg}") if $log
+      def failure(msg)
+        warn("Failure") { msg }
+        $log.warn("<AuditFailure> #{msg}") if $log
+      end
```

Comment:
@jrafanie I'm thinking we should just introduce a small non rails bit (changing database.yml and generate key) and make the rest just use standard rails. that would make this point moot.